k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapConnectionTest.java

```diff
@@ -275,6 +282,105 @@ public void open_authCramMd5WithoutAuthCramMd5Capability_shouldThrow() throws Ex
     }
 
     @Test
+    public void open_authXoauthWithSaslIr() throws Exception {
```

Comment:
please stick with the pattern the other methods in this test class use, it's very difficult to see where setup, test and assertions are otherwise